server/controllers/member/destroy.js

```diff
@@ -0,0 +1,10 @@
+const { Member } = require('../../models');
+
+module.exports = async ({ params: { id } }, res) => {
+    try {
+        let member = await Member.findByIdAndDelete(id);
+        res.send(member);
+    } catch (err) {
+        if (err) throw err
```

Comment:
It's not good enough to catch an error, you need to send an 500 (or whatever is appropriate) back to the client, otherwise the client will hang.